scripts/zalenium.sh

```diff
@@ -399,6 +399,7 @@ StartUp()
     fi
 
     echo "Copying files for Dashboard..."
+    sudo chown -R seluser.seluser  /home/seluser/videos
```

Comment:
I've never seen the dot syntax for chown before? Pretty sure it should be `seluser:seluser`, the man page says that too.
Also, this isn't safe to merge, because sudo is not always available, especially in OpenShift:
```
$ sudo chown -R seluser.seluser  /home/seluser/videos
sudo: PERM_SUDOERS: setresuid(-1, 1, -1): Operation not permitted
sudo: no valid sudoers sources found, quitting
sudo: setresuid() [0, 0, 0] -> [1000090000, -1, -1]: Operation not permitted
sudo: unable to initialize policy plugin
```

You'll need to do something like this:
```
if [ "${WE_HAVE_SUDO_ACCESS}" == "true" ]; then
    sudo chown -R seluser:seluser /home/seluser/videos
fi
```

Reply:
Thinking about it more, this more sounds like a fault with your kubernetes cluster.  When you create a persistent volume kuberentes should ensure the permissions are correct.  You shouldn't need sudo and changing permissions.